### PR TITLE
feat: Checkov skips and KMS encryption on CloudWatch log groups

### PR DESCRIPTION
**Pull request by patheard**
# Summary
* Add more policy skip comments for Checkov scan
* ~~Add KMS key encryption to CloudWatch log groups~~
* Add Terraform auto-format on save to the Devcontainer

# Expected changes
## Staging
* ~~Add KMS key ID to Lambda CloudWatch log groups~~
* None

## Prod
* None

Related #2

### CODE REVIEW

File: aws/backoff_retry_lambda/backoff_retry.tf [hunk 1]
```diff
@@ -37,5 +36,6 @@ resource "aws_lambda_event_source_mapping" "dead_letters" {
 
 resource "aws_cloudwatch_log_group" "backoff_log_group" {
   name              = "/aws/lambda/${aws_lambda_function.backoff_retry.function_name}"
+  kms_key_id        = data.aws_kms_key.cloudwatch.arn
```
Comment (CalvinRodo) [inline aws/backoff_retry_lambda/backoff_retry.tf:39]: We have no issues with managed KMS keys for this.

**Review by CalvinRodo — CHANGES_REQUESTED** [review]
See Comment.

File: aws/backoff_retry_lambda/backoff_retry.tf (hunk 1 above)
Comment (patheard) [reply aws/backoff_retry_lambda/backoff_retry.tf]: Done and done - reverted to default CloudWatch service key.

**Review by CalvinRodo — APPROVED** [review]
:taco::cat:

**Pull request merged**